package.json

```diff
@@ -37,16 +37,16 @@
     "@ephox/bedrock-client": "^13.0.0",
     "@ephox/bedrock-server": "^13.3.0",
     "@ephox/oxide-icons-tools": "^3.1.0",
-    "@ephox/swag": "^4.5.0",
-    "@tinymce/eslint-plugin": "^2.2.0",
+    "@ephox/swag": "^4.6.0-feature.20220914084517425.shaba4cd2f",
```

Comment:
I assume this is temporary for testing. Just pointing it out in case it was missed.

- [ ] Remove reference to feature branch before merging

Reply:
It's in the description/checklist already 😉 I need to wait for the swag version to be released before I can upgrade to it.
>  Stablize the swag dependency